rest/data/test.go

```diff
@@ -39,23 +39,6 @@ func (tc *DBTestConnector) FindTestsByTaskId(taskId, testId, testName, status st
 	if err != nil {
```

Comment:
maybe we should just return `testresult.Find(q)`?

Reply:
Fixed.